sigma/backends/netwitness/netwitness.py

```diff
@@ -0,0 +1,384 @@
+"""Module for the pySigma NetWitness backend"""
+
+import re
+from collections import defaultdict
+from typing import ClassVar, Dict, Optional, Pattern, Tuple, Union
+
+from sigma.conditions import (
+    ConditionAND,
+    ConditionFieldEqualsValueExpression,
+    ConditionItem,
+    ConditionNOT,
+    ConditionOR,
+    ConditionValueExpression,
+)
+from sigma.conversion.base import TextQueryBackend
+from sigma.conversion.deferred import DeferredQueryExpression
+from sigma.conversion.state import ConversionState
+from sigma.exceptions import SigmaConversionError
+from sigma.types import (
+    SigmaCompareExpression,
+    SigmaExpansion,
+    SigmaNumber,
+    SigmaRegularExpressionFlag,
+    SigmaString,
+    SpecialChars,
+)
+
+
+class NetWitnessBackend(TextQueryBackend):
+    """NetWitness backend."""
+
+    name: ClassVar[str] = "netwitness backend"
+    formats: ClassVar[Dict[str, str]] = {
+        "default": "Plain netwitness queries",
+    }
+    # TODO: does the backend requires that a processing pipeline is provided? This information can be used by user
+    # interface programs like Sigma CLI to warn users about inappropriate usage of the backend.
+    requires_pipeline: ClassVar[bool] = False
+
+    # Operator precedence: tuple of Condition{AND,OR,NOT} in order of precedence.
+    # The backend generates grouping if required
+    precedence: ClassVar[Tuple[ConditionItem, ConditionItem, ConditionItem]] = (ConditionNOT, ConditionAND, ConditionOR)
+    group_expression: ClassVar[Optional[str]] = (
+        "({expr})"  # Expression for precedence override grouping as format string with {expr} placeholder
+    )
+    parenthesize: bool = True
+
+    # Generated query tokens
+    token_separator: str = " "  # separator inserted between all boolean operators
+    or_token: ClassVar[Optional[str]] = "||"
+    and_token: ClassVar[Optional[str]] = "&&"
+    not_token: ClassVar[Optional[str]] = "NOT"
+    eq_token: ClassVar[Optional[str]] = " = "  # Token inserted between field and value (without separator)
```

Comment:
Isn't this with seperator?
```suggestion
    eq_token: ClassVar[Optional[str]] = " = "  # Token inserted between field and value (with separator)
```